docs/language/api.rst

```diff
@@ -436,6 +436,38 @@ Parameters may have the following keywords in front of them:
         => (zig-zag-sum 1 2 3 4 5 6)
         -3
 
+&kwonly
+    *(Python 3 only.)* Parameters that can only be called as
```

Comment:
We could turn this to a "Availability: Python 3" note like https://docs.python.org/3/library/os.html#os.ctermid